cpp/daal/include/algorithms/k_nearest_neighbors/kdtree_knn_classification_model.h

```diff
@@ -56,6 +56,26 @@ enum DataUseInModel
     doUse    = 1  /*!< The input data and labels will be the component of the trained kNN model */
 };
 
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__KDTREE_KNN_CLASSIFICATION__RESULTTOCOMPUTEID"></a>
+ * Available identifiers to specify the result to compute
+ */
+enum ResultToComputeId
+{
+    computeIndicesOfNeightbors = 0x00000001ULL,
```

Comment:
computeClassLabels might be added to skip class assignments

Reply:
Not sure that are you talking about. computeClassLabels is in resultToEvaluate, not resultToCompute